src/wp-includes/media.php

```diff
@@ -1871,6 +1875,10 @@ function wp_filter_content_tags( $content, $context = null ) {
 			if ( $filtered_iframe !== $match[0] ) {
 				$content = str_replace( $match[0], $filtered_iframe, $content );
 			}
+
+			// Unset iframe lookup to not run the same logic again unnecessarily if the same iframe tag is used more
+			// than once in the same blob of content.
```

Comment:
```suggestion
			/*
			 * Unset iframe lookup to not run the same logic again unnecessarily if the same iframe tag is used more
			 * than once in the same blob of content.
			 */
```

Same again because I am the worst.

:1234: It would also be good to fix in the tests